tests/bulk_create/tests.py

```diff
@@ -321,3 +321,19 @@ def test_ignore_conflicts_ignore(self):
         # Without ignore_conflicts=True, there's a problem.
         with self.assertRaises(IntegrityError):
             TwoFields.objects.bulk_create(conflicting_objects)
+
+    @skipUnlessDBFeature('can_return_rows_from_bulk_insert')
```

Comment:
Good point on only needing `bulk_create` on the affected instance. In the case of the test in which the parent is saved using `bulk_create` we need `skipUnlessDBFeature` since the parent won't get its primary key set and thus "bulk_create() prohibited to prevent data loss due to unsaved related object" will be raised when saving the child.